scripts/1-validate-packages.sh

```diff
@@ -10,7 +10,6 @@ validate () {
   query=$1;
   schema=$2;
   for file in $(find $PKG_DIR -name $query); do
-    echo "Validating $file";
```

Comment:
Mabye make this a `--verbose` instead of getting rid of it? I imagine it'll help quite a bit for debugging.